stdlib/3/faulthandler.pyi

```diff
@@ -0,0 +1,14 @@
+# Stubs for faulthandler (Python 3.7)
+
+from typing import IO, Union
+
+_File = Union[IO, int]
+
+def cancel_dump_traceback_later() -> None: ...
+def disable(): ...
+def dump_traceback(file: _File = ..., all_threads: bool = ...) -> None: ...
+def dump_traceback_later(timeout: int, repeat: bool = ..., file: _File = ..., exit: bool = ...) -> None: ...
+def enable(file: _File = ..., all_threads: bool = ...) -> None: ...
+def is_enabled() -> bool: ...
+def register(signum: int, file: _File = ..., all_threads: bool = ..., chain: bool = ...) -> None: ...
+def unregister(signum: int) -> None: ...
```

Comment:
Since the last two functions are not available on Windows, according to the documentation, they should be guarded like this:

```python
if sys.platform != "win32":
    ...
```